tests/Parse/MultilineStringSpec.hs

```diff
@@ -21,6 +21,11 @@ spec = do
         "normal string"
         "\"\"\"\nnormal string\"\"\""
 
+    it "crlf regression test" $ do
+      parse
+        "normal string"
+        "\"\"\"\r\nnormal string\"\"\""
```

Comment:
You should probably add the following tests

* A multiline string with a leading `\r` (missing `\n`) shouldn't be allowed
* `\r\n` should be allowed in the middle of a multi-string as well